File: Day - 8 (Dictionaries and Maps)/try.java
import java.util.HashMap;
import java.util.Map;
import java.util.Scanner;

public class PhoneBook {

    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        // Read input and assemble Phone Book
        int n = scanner.nextInt();
        Map<String, String> phoneBook = new HashMap<>();
        for (int i = 0; i < n; i++) {
            String[] contact = scanner.next().split(" ");
            phoneBook.put(contact[0], contact[1]);
        }

        // Process Queries
        while (scanner.hasNext()) {
            String name = scanner.next();
            if (phoneBook.containsKey(name)) {
                System.out.println(name + "=" + phoneBook.get(name));
            } else {
                System.out.println("Not found");
            }
        }

        scanner.close();
    }
}

File: Day - 8 (Dictionaries and Maps)/try.c
#include <stdio.h>
#include <string.h>

int main() {
    // Read input and assemble Phone Book
    int n;
    scanf("%d", &n);
    char phoneBook[1000][20]; // Assuming names and phone numbers are at most 20 characters long
    for (int i = 0; i < n; i++) {
        scanf("%s", phoneBook[i]);
        scanf("%s", phoneBook[i] + strlen(phoneBook[i]) + 1);
    }

    // Process Queries
    char name[20];
    while (scanf("%s", name) == 1) {
        int found = 0;
        for (int i = 0; i < n; i++) {
            if (strcmp(name, phoneBook[i]) == 0) {
                printf("%s=%s\n", name, phoneBook[i] + strlen(phoneBook[i]) + 1);
                found = 1;
                break;
            }
        }
        if (!found) {
            printf("Not found\n");
        }
    }

    return 0;
}
